WIP: integration of CI system

hoehnp opened the pull request.

hoehnp committed:
added circleci template from own repository

	new file:   .circleci/config.yml

hoehnp committed:
change path for game.jar

	modified:   .circleci/config.yml

hoehnp committed:
integration of files from own repo

	modified:   .circleci/config.yml
	new file:   .gitignore
	new file:   game/starter/exe/bolzplatz2006/Makefile
	new file:   game/starter/exe/bolzplatz2006/bolzplatz2006.layout
	new file:   game/starter/exe/bolzplatz2006/bolzplatz2006_private.h
	new file:   game/starter/exe/bolzplatz2006/bolzplatz2006_private.rc
	new file:   game/starter/exe/bolzplatz2006/bolzplatz2006_private.res
	new file:   game/starter/exe/settings/Makefile
	new file:   game/starter/exe/settings/bolzplatz2006_private.h
	new file:   game/starter/exe/settings/bolzplatz2006_private.rc
	new file:   game/starter/exe/settings/bolzplatz2006_private.res
	new file:   game/starter/exe/settings/settings.layout
	new file:   game/starter/exe/stadioneditor/Makefile
	new file:   game/starter/exe/stadioneditor/bolzplatz2006_private.h
	new file:   game/starter/exe/stadioneditor/bolzplatz2006_private.rc
	new file:   game/starter/exe/stadioneditor/bolzplatz2006_private.res
	new file:   game/starter/exe/stadioneditor/stadioneditor.layout
	new file:   game/starter/exe/teameditor/Makefile
	new file:   game/starter/exe/teameditor/bolzplatz2006_private.h
	new file:   game/starter/exe/teameditor/bolzplatz2006_private.rc
	new file:   game/starter/exe/teameditor/bolzplatz2006_private.res
	new file:   game/starter/exe/teameditor/teameditor.layout

hoehnp committed:
fixed encoding error in build.xml

	modified:   game/build.xml

hoehnp committed:
test installation of wine

	modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp commented:
this will close #2

hoehnp committed:
added wine-gecko

	modified:   .circleci/config.yml

hoehnp committed:
add files from Inno Setup to generate setup package
	modified:   .circleci/config.yml
	new file:   tools/setup/v1.0-windows/ISCC.exe
	new file:   tools/setup/v1.0-windows/ISCmplr.dll
	new file:   tools/setup/v1.0-windows/ISPP.dll
	new file:   tools/setup/v1.0-windows/isbunzip.dll
	new file:   tools/setup/v1.0-windows/isbzip.dll
	new file:   tools/setup/v1.0-windows/islzma.dll
	new file:   tools/setup/v1.0-windows/isscint.dll
	new file:   tools/setup/v1.0-windows/isunzlib.dll
	new file:   tools/setup/v1.0-windows/iszlib.dll

hoehnp committed:
remove non-existing wine-gecko package

	modified:   .circleci/config.yml

hoehnp committed:
corrected path in bolzplatz2006-install.iss

	modified:   tools/setup/v1.0-windows/bolzplatz2006-install.iss

hoehnp committed:
corrected more path in bolzplatz2006-install.iss

	modified:   tools/setup/v1.0-windows/bolzplatz2006-install.iss

hoehnp committed:
further corrected pathes in bolzplatz2006-install.iss

	modified:   tools/setup/v1.0-windows/bolzplatz2006-install.iss

hoehnp committed:
added missing images

	new file:   tools/setup/v1.0-windows/WizModernImage-IS.bmp
	new file:   tools/setup/v1.0-windows/WizModernImage.bmp
	new file:   tools/setup/v1.0-windows/WizModernSmallImage-IS.bmp
	new file:   tools/setup/v1.0-windows/WizModernSmallImage.bmp

hoehnp committed:
adding more missing files from Inno Setup

	new file:   tools/setup/v1.0-windows/ISPPBuiltins.iss
	new file:   tools/setup/v1.0-windows/Setup.e32
	new file:   tools/setup/v1.0-windows/SetupLdr.e32

hoehnp committed:
added translation files for Inno Setup

	new file:   tools/setup/v1.0-windows/Languages/French.isl
	new file:   tools/setup/v1.0-windows/Languages/German.isl

hoehnp committed:
added missing Default.isl file
	new file:   tools/setup/v1.0-windows/Default.isl

hoehnp committed:
copy starter exe to folder for Inno Setup

	modified:   .circleci/config.yml

hoehnp committed:
correct name of compiled exe

	modified:   .circleci/config.yml

hoehnp committed:
corrected pathfor bolzplatz2006.exe

	modified:   .circleci/config.yml

hoehnp committed:
check correct path

	modified:   .circleci/config.yml

hoehnp committed:
check for right path

	modified:   .circleci/config.yml

hoehnp committed:
use cp with hopefully correct path now

	modified:   .circleci/config.yml

hoehnp committed:
add einstellungen.exe for Inno Setup

	modified:   .circleci/config.yml

hoehnp committed:
correct file name while copying

	modified:   .circleci/config.yml

hoehnp committed:
test commit with pathes only modified in bolzplatz2006-install.iss

	modified:   tools/setup/v1.0-windows/bolzplatz2006-install.iss

hoehnp committed:
temporary disabled option to just check if file works

hoehnp committed:
corrected artifact file

	modified:   .circleci/config.yml

Xenoage commented:
Before merging, I want to make sure that I can perform the build process, too. Can you provide step-by-step instructions which software is required and what has to be done?

hoehnp commented:
No problem at all. All steps are defined in [.circelci/config.yml](https://github.com/hoehnp/Bolzplatz2006/blob/master/.circleci/config.yml)

hoehnp committed:
test large inclusion of files according Issue on github

	modified:   .circleci/config.yml
	modified:   tools/setup/v1.0-windows/bolzplatz2006-install.iss

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
copy javaw.exe to setup folder

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
include complete adoptOpenJDK JRE
	modified:   .circleci/config.yml

hoehnp committed:
set UsePreviousAppDir to no

	modified:   tools/setup/v1.0-windows/bolzplatz2006-install.iss

hoehnp committed:
corrected filename to git version

	modified:   .circleci/config.yml

hoehnp committed:
try to correct path for copying

	modified:   .circleci/config.yml

hoehnp committed:
further testing cp
	modified:   .circleci/config.yml

hoehnp committed:
added dom4j to lib dir

	modified:   .circleci/config.yml

hoehnp committed:
corrected path for dom4j
	modified:   .circleci/config.yml

hoehnp committed:
corrected flag for wget

	modified:   .circleci/config.yml

hoehnp committed:
corrected download link for dom4j

	modified:   .circleci/config.yml

hoehnp committed:
changed target in lwjgl to version 11
	modified:   libs/lwjgl/build.xml

hoehnp committed:
activate building of lwjgl

	modified:   .circleci/config.yml

hoehnp committed:
download java bindings for sdl

	modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
added download for sdljava

	modified:   .circleci/config.yml

hoehnp committed:
try to fix sdljava

	modified:   .circleci/config.yml

hoehnp committed:
correct path for sdljava

	modified:   .circleci/config.yml

hoehnp committed:
further corrections of path

	modified:   .circleci/config.yml

hoehnp committed:
added license file

	modified:   .circleci/config.yml

hoehnp committed:
add binaries from lwjgl
	modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
corrected typoe in unzip command

	modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
trying to compile irrlicht-0.14.0 for windows

hoehnp committed:
test building irrlicht-0.14.0

	modified:   .circleci/config.yml

hoehnp committed:
modified:   libs/archive/irrlicht-0.14-patched/include/matrix4.h

hoehnp committed:
modified:   libs/archive/irrlicht-0.14-patched/CGUIButton.cpp

hoehnp committed:
corrected path from windows to linux in Makefile.win

	modified:   libs/archive/irrlicht-0.14-patched/Makefile.win

hoehnp committed:
modified:   libs/archive/irrlicht-0.14-patched/Makefile.win

hoehnp committed:
modified:   .circleci/config.yml
	modified:   libs/archive/irrlicht-0.14-patched/CGUIButton.cpp
	modified:   libs/archive/irrlicht-0.14-patched/include/quaternion.h

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   libs/archive/irrlicht-0.14-patched/CGUIButton.cpp

hoehnp committed:
modified:   libs/archive/irrlicht-0.14-patched/CGUIButton.cpp

hoehnp committed:
further patches for irrlicht

hoehnp committed:
new file:   libs/archive/irrlicht-0.14-patched/bin/.gitkeep

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
added swig package

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
modified:   .circleci/config.yml

hoehnp committed:
add Irrlicht.dll to installation folder
	modified:   .circleci/config.yml

hoehnp committed:
tried to merge

Xenoage committed:
Website, including user mods

Xenoage committed:
Ignore IntelliJ files

hoehnp committed:
Merge pull request #4 from Xenoage/master

added some changes to website from upstream

hoehnp committed:
Update config.yml

hoehnp committed:
Update config.yml

hoehnp committed:
Update config.yml

hoehnp committed:
Update config.yml

hoehnp committed:
Update config.yml

hoehnp committed:
Update config.yml

hoehnp committed:
Update build.xml

hoehnp committed:
Update build.xml

hoehnp committed:
Update build.xml

hoehnp committed:
fixed issue in irrlicht.jar

hoehnp committed:
Update build.xml

hoehnp committed:
Update config.yml

hoehnp committed:
Update config.yml

hoehnp committed:
Update config.yml

hoehnp committed:
Update bolzplatz2006-install.iss

hoehnp committed:
Update README.md

hoehnp committed:
Update config.yml

hoehnp committed:
updates to config.yml for linux build

hoehnp committed:
correct mkdir

hoehnp committed:
intermediate commit

hoehnp committed:
revert build.xml

hoehnp committed:
fixed typos in directory names

hoehnp committed:
fixed typo

hoehnp committed:
further typos

hoehnp committed:
added artifact for linux

hoehnp committed:
add second artifact

hoehnp committed:
fix second artifact

hoehnp committed:
added compilation of irrlicht

hoehnp committed:
added build-essentials

hoehnp committed:
try to fix error from previous build

hoehnp committed:
new fixes for build errors

hoehnp committed:
added missing dependency

hoehnp committed:
fixing dependencies

hoehnp committed:
furter attempts to fix includes

hoehnp committed:
added jirr binaries
added more files to setup and circleci
